ansible/roles/ocp4-workload-integreatly/tasks/post_workload.yml

```diff
@@ -2,7 +2,47 @@
 # Implement your Post Workload deployment tasks here
 # --------------------------------------------------
 
+- name: Get current htpasswd secret
+  shell: oc get secret htpasswd-secret -n openshift-config -o json | jq -j '.data.htpasswd' | base64 --decode > /tmp/htpasswd
 
+- name: Backup htpasswd
+  shell: cp /tmp/htpasswd /tmp/htpasswd.backup
+
+- name: Generate dedicated admin password
+  shell: openssl rand -base64 12
+  register: _gen_password
+
+- name: Set dedicated admin password
+  set_fact:
+    _dedicated_admin_password: "{{ _gen_password.stdout }}"
+
+- name: Create dedicated admin user
+  shell: htpasswd -B -b /tmp/htpasswd {{ ocp4_workload_integreatly.dedicated_admin_username }} {{ _dedicated_admin_password }}
+
+- name: Delete current htpasswd secret
```

Comment:
@ciaranRoche Rather than deleting/creating the secret, would it be possible to run `oc replace` or something similar? Reason I ask is that in the RHMI 1.x ansible installer we've run delete/create commands and they can cause problems. For example, failed create commands after removing the resource can leave the cluster in a bad state.